src/main/java/hello/HelloController.java

```diff
@@ -1,16 +1,28 @@
 package hello;
 
-import io.micrometer.core.annotation.Timed;
-import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
+import org.springframework.web.bind.annotation.RequestMapping;
+
+import javax.servlet.http.HttpServletRequest;
 
 @RestController
-@Timed
-public class HelloController {
-    
+public class HelloController implements Comparable {
+    public static String FOO_VAR = "";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "FOO_VAR" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.35.192.52.128.xip.io/coding_rules#rule_key=squid%3AS3008)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this "public static FOO_VAR" field final [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.35.192.52.128.xip.io/coding_rules#rule_key=squid%3AS1444)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make FOO_VAR a static final constant or non-public and provide accessors if needed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.35.192.52.128.xip.io/coding_rules#rule_key=squid%3AClassVariableVisibilityCheck)